release/platform/platform.go

```diff
@@ -38,9 +38,9 @@ const (
 	// While arm64 and aarch64 are the same architecture, some Linux distros
 	// use arm64 (Debian and RHEL) and others use aarch64 (Amazon 2).
 	ArchAarch64 Arch = "aarch64"
-	ArchS390x        = "s390x"
-	ArchPpc64le      = "ppc64le"
-	ArchX86_64       = "x86_64"
+	ArchS390x   Arch = "s390x"
```

Comment:
[nit] I think only have the first element typed is a more idiomatic way inside Golang such as 
```
const (
	ArchArm64 Arch = "arm64"
        ArchAarch64 = "aarch64"
        ArchS390x  = "s390x"
        ...
```

Reply:
That only works for `iota`, I think. Example: https://go.dev/play/p/IAbrnm9DkIf

Reply:
We should probably do the same thing with all these other constants (OS, Pkg, Repo), which have the same problem!

Reply:
@mmcclimon Fixed in 13a93d2a.